src/modules/assistants/icons/AssistantBaseIcon.tsx

```diff
@@ -46,7 +42,7 @@ export function AssistantBaseIcon({
       data-size={size}
     >
       <AssistantIconColor color={color}>
-        <>{Icon ? <Icon /> : initialLetter}</>
+        <Icon />
```

Comment:
I wouldn't remove the icon fallback to initial letter, unless it's requested from the design.